sonar-orchestrator/src/test/java/com/sonar/orchestrator/OrchestratorBuilderTest.java

```diff
@@ -132,6 +132,20 @@ public void enable_default_force_authentication() throws Exception {
     assertThat(properties.getProperty("sonar.forceAuthentication")).isNull();
   }
 
+  @Test
+  public void empty_sonar_properties() throws Exception {
```

Comment:
Perhaps it makes sense to add an UT with sonar properties set?

Reply:
all others UTs will generate `sonar.properties`